File: ZV_CursusAdministratie.Web/ZVCursusAdministratieWeb/src/app/api/models/Cursus.ts
import {CursusInstantie} from "./CursusInstantie";

export class Cursus {
    constructor(public id: number, public duur:number, public titel: string, public code: string, public cursusinstanties: CursusInstantie[]){}
    
    // id: number;
    // duur: number;
    // titel: string;
    // code: string;
    // cursusinstanties: CursusInstantie[];
}

File: ZV_CursusAdministratie/Entities/Cursus.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace ZV_CursusAdministratie.Entities
{
    public class Cursus
    {
        public int Id { get; set; }
        public string Titel { get; set; }
        public string Code { get; set; }

        public int Duur { get; set; }

        public List<CursusInstantie> CursusInstanties { get; set; }

    }
}